packages/plugins/i18n/server/constants/iso-locales.json

```diff
@@ -451,10 +451,26 @@
     "code":"en-AS",
     "name":"English (American Samoa) (en-AS)"
   },
+  {
+    "code":"en-AI",
+    "name":"English (Anguilla) (en-AI)"
```

Comment:
language code: `aig`
documentation: https://iso639-3.sil.org/code/aig

Reply:
my bad, Anguilla is good with this format!

Reply:
@ronronscelestes  Yes, it is. Anguilla is different from Antigua